.github/workflows/tests.yaml

```diff
@@ -0,0 +1,110 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+env:
+  BEE_VERSION: '1.5.1-d0a77598-stateful'
+  BLOCKCHAIN_VERSION: '1.2.0-for-1.5.1-d0a77598-stateful'
+  BEE_ENV_PREFIX: 'swarm-test'
+  BEE_IMAGE_PREFIX: 'docker.pkg.github.com/ethersphere/bee-factory'
+  COMMIT_VERSION_TAG: 'false'
+  WORKERS: 4
+  BEE_API_URL: 'http://127.0.0.1:1633'
+  BEE_PEER_API_URL: 'http://127.0.0.1:11633'
+  BEE_DEBUG_API_URL: 'http://127.0.0.1:1635'
+  BEE_PEER_DEBUG_API_URL: 'http://127.0.0.1:11635'
+  BEE_TEST_CHEQUEBOOK: true
+
+jobs:
+  nodejs:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
well, there is no long time support for node v12 already, but we should support for 18.x

Reply:
ok, I removed node 12, but not added 18. Due to https://github.com/fairDataSociety/fdp-storage/issues/6